src/utils/commentManager.ts

```diff
@@ -1,7 +1,7 @@
 import * as vscode from "vscode";
 
 import { addToCache, getFromCache } from "./addonCache";
-import getCommentLocation, {
+import getThreadLocation, {
```

Comment:
It's atypical (but not unread of) for an ES module to have both named exports and a default export. The default export of a Node module is typically the namespace on which other functions live. The use of a default export is also more typical in CommonJS modules. For example, Node's built in File System module has a default export that exposes all the various methods (`const fs = require('fs')` in CJS, `import fs from 'node:fs'`). See [this post](https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad) for more on why some devs tend to avoid default exports in ESM.

I'd suggest simply making `getThreadLocation` a named export.